doc/nrf/introduction.rst

```diff
@@ -8,20 +8,20 @@ About the |NCS|
    :depth: 2
 
 The |NCS| enables you to develop applications for nRF52, nRF53, and nRF91 Series devices.
-It is a set of open source projects maintained by Nordic Semiconductor, consisting of several Git repositories publicly hosted on `GitHub`_:
+The SDK consists of a combination of Nordic-developed software and open source projects, hosted as `Git_` repositories in the `nrfconnect GitHub organization`_. Some of the more prominent repositories include:
```

Comment:
'It is a combination of software developed by Nordic Semiconductor and open source projects, hosted...'